<launch>
  <!-- Start tilt joint controller -->
  <rosparam file="$(find jeeves_neck)/config/neck_config.yaml" command="load"/>
  <node name="controller_spawner" pkg="dynamixel_controllers" type="controller_spawner.py"
    args="--manager=dxl_manager
          --port dxl_USB0
          tilt_right_controller
          tilt_left_controller
          pan_controller
          "
    output="screen" />

  <!-- Start joints trajectory controller -->
  <rosparam file="$(find jeeves_neck)/config/joints_trajectory_controller.yaml" command="load" />
  <node name="controller_spawner_meta" pkg="dynamixel_controllers" type="controller_spawner.py"
    args="--manager=dxl_manager
          --type=meta
          neck_controller
          tilt_right_controller
          tilt_left_controller
          pan_controller
          "
    output="screen" />
</launch>
